src/Pages/Signup/index.js

```diff
@@ -1,7 +1,258 @@
+      .fill()
+      .map((v, i) => i + 1);
+
+    return (
+      <main className="signup">
+        <div className="signupContainer">
+          <section className="header">
+            <h1 className="logo">KaKao</h1>
+          </section>
+          <section className="main">
+            <div className="mainWrap">
+              <h2 className="mainTitle">카카오계정 정보를 입력해 주세요.</h2>
+              <form className="mainForm" onSubmit={this.handleSubmit}>
+                <div className="email">
+                  <strong className="emailTitle">카카오계정 이메일</strong>
+                  <div className="emailBox">
+                    <label className="emailLabel">
+                      <input
+                        autoComplete="off"
+                        name="email"
+                        className="emailInput"
+                        type="text"
+                        placeholder="이메일 입력"
+                        onChange={this.handleInput}
+                      />
+                    </label>
+                  </div>
+                  <ul className="emailCautionLists">
+                    <li className="emailCautionList">
+                      ・ 입력한 카카오메일로 카카오계정에 로그인할 수 있습니다.
+                    </li>
+                    <li className="emailCautionList">
+                      ・ 한번 만든 카카오메일은 변경할 수 없으니, 오타가 없도록
+                      신중히 확인해 주세요.
+                    </li>
+                    <li className="emailCautionList">
+                      ・ 이메일은 @포함, 비밀번호는 5글자 이상 작성 형식을
+                      지켜주세요.
+                    </li>
+                    <li className="emailCautionList">
+                      ・ 생성한 카카오메일로 카카오계정과 관련한 알림을 받아볼
+                      수 있습니다.
+                    </li>
+                  </ul>
+                </div>
+                <div className="pw">
+                  <strong className="pwTitle">비밀번호</strong>
+                  <div className="pwBox">
+                    <label htmlFor="pwInput" className="pwLabel">
+                      <input
+                        autoComplete="off"
+                        className="pwInput"
+                        type="password"
+                        name="password"
+                        placeholder="비밀번호(8~32자리)"
+                        onChange={this.handleInput}
+                      />
+                    </label>
+                  </div>
+                </div>
+                <div className="nickname">
+                  <strong className="nicknameTitle">닉네임</strong>
+                  <div className="nicknameBox">
+                    <label htmlFor="nicknameInput" className="nicknameLabel">
+                      <input
+                        autoComplete="off"
+                        className="nicknameInput"
+                        type="text"
+                        name="nickname"
+                        placeholder="닉네임을 입력해 주세요."
+                        onChange={this.handleInput}
+                      />
```

Comment:
```suggestion
                    <label className="nicknameLabel">
                      <input
                        autoComplete="off"
                        className="nicknameInput"
                        type="text"
                        name="nickname"
                        placeholder="닉네임을 입력해 주세요."
                        onChange={this.handleInput}
                      />
```
이 경우는 htmlFor가 불필요해보입니다. 비슷한 다른 케이스에도 적용해주세요